src/runtime/rpc/rpc_session.cc

```diff
@@ -1190,23 +1192,42 @@ void RPCSession::EventHandler::HandlePackedCall() {
   CHECK_EQ(state_, kRecvCode);
 }
 
-PackedFunc WrapTimeEvaluator(PackedFunc pf, TVMContext ctx, int number, int repeat) {
-  auto ftimer = [pf, ctx, number, repeat](TVMArgs args, TVMRetValue *rv) {
+PackedFunc WrapTimeEvaluator(PackedFunc pf,
+                             TVMContext ctx,
+                             int number,
+                             int repeat,
+                             int min_repeat_ms) {
+  auto ftimer = [pf, ctx, number, repeat, min_repeat_ms](TVMArgs args, TVMRetValue *rv) {
     TVMRetValue temp;
     std::ostringstream os;
     // skip first time call, to activate lazy compilation components.
     pf.CallPacked(args, &temp);
     DeviceAPI::Get(ctx)->StreamSync(ctx, nullptr);
+    int dynamic_number = number;
```

Comment:
You used to modify `number` directly which had a nice property of remembering the suitable value of `number` between runs. I think you can still achieve this effect by declaring the lambda as `mutable` (won't be thread-safe though, so I'm not sure).

Reply:
Sorry that my previous conclusion about tvm packed func is wrong. It works well with vanilla capture-by-reference. So I turn it back and tests are also added.